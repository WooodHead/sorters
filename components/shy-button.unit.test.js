import ShyButton from './shy-button'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('shy-button', () => {
    it('shallow-renders', () => {
        const component = shallow(<ShyButton>âœŽ</ShyButton>)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
