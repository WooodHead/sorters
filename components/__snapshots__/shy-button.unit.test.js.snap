// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shy-button shallow-renders 1`] = `
<a
  style={
    Object {
      "fontSize": "140%",
    }
  }
>
  âœŽ
</a>
`;
